YOLONAS-POSE.md

```diff
@@ -0,0 +1,110 @@
+# YOLO-NAS-POSE
+### A Next-Generation, Pose Estimation Foundational Model generated by Deci’s Neural Architecture Search Technology
+
+Deci is thrilled to announce the release of a new object detection model, YOLO-NAS-POSE - a successor of [YOLO-NAS](YOLONAS.md), 
```

Comment:
is it a successor? I would say a derivative

Reply:
Fixed in f4bc27730ea7141b7f07ea493d68bc8e61078776